Sources/BrowserServicesKit/Common/Extensions/BundleExtension.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  BundleExtension.swift
+//  DuckDuckGo
+//
+//  Copyright © 2021 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+extension Bundle {
+    
+    struct Keys {
+        static let versionNumber = "CFBundleShortVersionString"
+    }
+    
+    var releaseVersionNumber: String? {
+        return infoDictionary?[Keys.versionNumber] as? String
```

Comment:
Note that this returns version number without the build. Just bringing this up as I've seen tests using both version and build number.